torch/export/_trace.py

```diff
@@ -1920,7 +1996,7 @@ def _produce_guards_callback(gm):
             new_fake_kwargs,
             new_fake_constant_attrs,
             map_fake_to_real,
-        ), _fakify_module_inputs(fake_args, fake_kwargs, fake_mode):
+        ), _fakify_module_inputs(fake_args, fake_kwargs, fake_mode), _override_builtin_min_max():
```

Comment:
surprised you're not getting a lint warning here, is this the right way to format multiple contexts?